client/catapult/extensions/mongo/src/mappers/MapperInclude.h

```diff
@@ -21,16 +21,7 @@
 
 #pragma once
 #include "catapult/types.h"
-
-namespace bsoncxx {
-	inline namespace v_noabi {
-		namespace array { class view; }
-		namespace document {
-			class value;
-			class view;
-		}
-		namespace types { struct b_binary; }
-	}
-}
-
-namespace mongocxx { inline namespace v_noabi { class cursor; } }
+#include <bsoncxx/builder/stream/array-fwd.hpp>
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/types-fwd.hpp>
+#include <mongocxx/cursor-fwd.hpp>
```

Comment:
Should we remove MapperInclude.h or leave this?

Reply:
i think it is ok to leave